loama/src/router/index.ts

```diff
@@ -14,9 +17,25 @@ const router = createRouter({
             component: LoginView
         },
         {
-            path: `/home/:filePath(.*)`,
-            name: 'home',
-            component: HomeView
+            path: "/",
```

Comment:
Minor remark: the URIs http://localhost:5173/inbox and http://localhost:5173/request redirect to http://localhost:5173/home/. Can you add their paths?